testing gen file for release notes (#7817)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/dbt-cloud-release-notes-gen.md

---

> title: "Weekly dbt release notes"

Reviewer: Even though this is hidden, it's still possible someone finds it through some path - I think the title and the sidebar should specify these are single tenant release notes

> title: "Weekly dbt release notes"

Author: ```suggestion
title: "Weekly dbt single-tenant release notes"
```

> description: "Weekly dbt release notes"

Reviewer: Something like this so they're distinguishable from the MT release notes for folks who are viewing them
```suggestion
description: "Release notes for weekly single-tenant updates."
```

id: "dbt-cloud-release-notes-gen"

> sidebar: "dbt release notes"

Reviewer: Even thought it's hidden, this will appear on the sidebar when the user is on the page, we might want to avoid using language that could cause them to be confused with the regular MT release notes
```suggestion
sidebar: "dbt single-tenant release notes"
```

pagination_next: null
pagination_prev: null
unlisted: true
---

> <Constant name="cloud" /> release notes for recent and historical changes. Release notes fall into one of these categories:

Reviewer: Not sure "packages" is the right term, but something like that so it clarifies is per update bundle/version/etc. 
```suggestion
<Constant name="cloud" /> Single-tenant release notes for weekly updates. Release notes fall into one of these categories:
```
